identity-iota-core/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "identity-iota-core"
+version = "0.5.0-dev.4"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+homepage = "https://www.iota.org"
+keywords = ["iota", "tangle", "identity"]
+license = "Apache-2.0"
+readme = "../README.md"
+repository = "https://github.com/iotaledger/identity.rs"
+description = ""
+
+[dependencies]
+identity-core = { version = "=0.5.0-dev.4", path = "../identity-core" }
+identity-did = { version = "=0.5.0-dev.4", path = "../identity-did" }
+lazy_static = { version = "1.4", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["std", "derive"] }
+strum = { version = "0.21", features = ["derive"] }
+thiserror = { version = "1.0", default-features = false }
+
+[dependencies.iota-client]
+git = "https://github.com/iotaledger/iota.rs"
+rev = "7f5ae8f5b27f5948d8e1b4717419cc1321d862da"
+features = ["tls"]
+default-features = false
```

Comment:
The `identity-iota-core` crate should not need `iota-client` at all. If the only reason for it is `MessageId` we can import `bee-message` directly instead, which `iota.rs` just re-exports.